libcoap: server.c example does not compile

I tried to simply compile the sample gnrc_networking + the package libcoap (without editing something in main.c), I receive:

```
RIOT-2016.02/sys/posix/sockets/posix_sockets.c:184:12: error: '_implicit_bind' defined but not used [-Werror=unused-function]
 static int _implicit_bind(socket_t *s, void *addr)
            ^
cc1: all warnings being treated as errors
```

--- a/Makefile.dep
+++ b/Makefile.dep
@@ -3,6 +3,7 @@ include $(RIOTBASE)/drivers/Makefile.dep
 
 ifneq (,$(filter libcoap,$(USEPKG)))
     USEMODULE += posix_sockets
+    USEMODULE += gnrc_conn_udp
 endif
 
 ifneq (,$(filter nhdp,$(USEMODULE)))

--- a/sys/posix/sockets/posix_sockets.c
+++ b/sys/posix/sockets/posix_sockets.c
@@ -31,6 +31,16 @@
 #include "sys/socket.h"
 #include "netinet/in.h"
 
+#ifdef  MODULE_CONN_IP
+#   include "net/conn/ip.h"
+#endif  /* MODULE_CONN_IP */
+#ifdef  MODULE_CONN_TCP
+#   include "net/conn/tcp.h"
+#endif  /* MODULE_CONN_TCP */
+#ifdef  MODULE_CONN_UDP
+#   include "net/conn/udp.h"
+#endif  /* MODULE_CONN_UDP */
+
 #define SOCKET_POOL_SIZE    (4)
 
 /**

